bridges/otelzap/core.go

```diff
@@ -142,7 +150,15 @@ func (o *Core) Sync() error {
 // Check determines whether the supplied Entry should be logged using core.Enabled method.
 // If the entry should be logged, the Core adds itself to the CheckedEntry and returns the result.
 func (o *Core) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
-	if o.Enabled(ent.Level) {
```

Comment:
With this, the `Enabled()` method isn't being used anymore within the bridge. And it's not required by `zapcore.Core`. Should we remove it?

Reply:
Makes sense. @khushijain21 any reason why it was introduced?

Reply:
`Core` implements `LevelEnabler` which implements the `Enabled` method. We won't be able to remove this to comply with the interface. Since we are reusing the logic from `Enabled`, the method becomes redundant

Reply:
Maybe we can add this 

```
if ent.LoggerName != "" && logger.Enabled(context.Background(),r) {
		return ce.AddCore(ent, o)
}
else if o.Enabled(ent.Level) {
		return ce.AddCore(ent, o)
}
return ce
```

Reply:
I see. Needed by https://pkg.go.dev/go.uber.org/zap/zapcore#Core